packages/babel-helpers/src/helpers.js

```diff
@@ -478,6 +476,11 @@ helpers.construct = helper("7.0.0-beta.0")`
       return false;
     }
   }
+`;
+
+helpers.construct = helper("7.0.0-beta.0")`
```

Comment:
This one should be `7.0.1` too?

Reply:
No, because this helper was already available. I only restructured its internals.